app/models/setting/email.rb

```diff
@@ -0,0 +1,19 @@
+class Setting::Email < Setting
+  def self.load_defaults
+    return unless super
+    smtp_options = ActionMailer::Base.smtp_settings || {}
+    method = ActionMailer::Base.delivery_method
+    self.transaction do
+      [
+        self.set('email_yaml', N_("Uses email.yaml configurations instead"), false, N_('YAML conifguration')),
```

Comment:
1) typo in the setting name
2) "configurations" reads poorly
3) "instead" of what?